docs/data-modeling.md

```diff
@@ -0,0 +1,66 @@
+
+#### Column
+A column is a fundamental data element and does not need to be broken down any further. Each record is composed of one or more columns. Each column has a data type. For details about the data type, refer to [Data-type mapping between ScalarDB and other databases](https://scalardb.scalar-labs.com/docs/latest/schema-loader#data-type-mapping-between-scalardb-and-other-databases).
+
+#### Index
+An index is a copy of the records in a single table, sorted by a column. ScalarDB queries use indexes to find data more efficiently in a table, given the values of a particular column. 
+
+
+### How to locate records
+This section discusses how to locate records from a table.
+
+#### Primary key
+A primary key uniquely identifies each record; no two records can have the same primary key. Therefore, you can locate a record by specifying a primary key. A primary key comprises a set of partition keys and clustering keys.
+
+#### Partition key
+A partition key uniquely identifies a partition. A partition key comprises a set of columns, which are called partition key columns. When you specify only a partition key, you can get a set of records that belong to the partition.
+
+#### Clustering key
+A clustering key uniquely identifies a record within a partition. It comprises a set of columns called clustering key columns. When you want to specify a clustering key, you should specify a partition key for efficient lookups. When you specify a clustering key without a partition key,
+you end up scanning all the partitions if you don’t have proper indexes. Scanning all the partitions is very time-consuming, so please do it only when you know what you are doing.
+
+Records within a partition are assumed to be sorted by clustering key columns. Therefore, you can specify a part of clustering key columns in the defined order to narrow down the results to be returned. 
+
+#### Index key
+An index key identifies records by looking up the key in indexes.
+
+## How to design your database schemas
+You can design your database schemas similarly to the relational model, but there are a few best practices to follow.
+
+### Care about data distribution
+It would be best if you tried to balance loads to partitions by properly selecting partition and clustering keys. For example, in a banking application, if you choose an account ID as a partition key, you can perform any account operations for a specific account within the partition to which the account belongs. So, if you operate on different account IDs, you access different partitions. On the other hand, if you choose a branch ID as a partition key and an account ID as a clustering key, all the accesses to a branch’s account IDs go to the same partition. In addition, you should choose a high cardinality column as a partition key because creating a small number of large partitions causes an imbalance in loads and data sizes.
```

Comment:
> all the accesses to a branch’s account IDs go to the same partition.

Are you writing this part to say that it's not good (for most cases)?  If so, it would be nice if it could be more direct.